english/coffee/nude-run/warm-water/surrender/surrender.md

```diff
@@ -9,11 +9,17 @@ you are sad.
 
 "Close your eyes, and feel, see, those bees inside you."
 
-As he said that, a warm feeling started radiating from my
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.